src/core/store/modules/partner/partner.reducer.ts

```diff
@@ -0,0 +1,21 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { PartnerState } from '@types';
+
+import { PartnerActions } from './partner.actions';
+
+export const partnerInitialState: PartnerState = {
+  id: undefined,
+  address: undefined,
+};
```

Comment:
This doesn't seem needed, how about we just do the following below
```typescript
createReducer({}, (builder) => {})
```

Reply:
Yeah, agree with you here on better using `null`. We can transition to this approach in a refactor for all the state so we have a convention.

Will leave as is for now.